src/emailservice/email_server.rb

```diff
@@ -12,11 +12,40 @@
 
 post "/send_order_confirmation" do
   data = JSON.parse(request.body.read, object_class: OpenStruct)
-  Pony.mail(
-    to:       data.email,
-    from:     "noreply@example.com",
-    subject:  "Your confirmation email",
-    body:     erb(:confirmation, locals: { order: data.order }),
-    via:      :logger
-  )
+
+  # get the current auto-instrumented span
+  current_span = OpenTelemetry::Trace.current_span
+  current_span.add_attributes({
+    "app.email.order_id" => data.order.order_id,
```

Comment:
What generates the order ID? imo this should be normalized across services; app.order.id perhaps?

Reply:
app.order.id sounds good!
That's the name used in the [demo.pb](https://github.com/open-telemetry/opentelemetry-demo-webstore/blob/7c877d652ddb89070c9465b3f99fe17a1ea62a2c/src/shippingservice/genproto/hipstershop/demo.pb.go#L1406)

Will update it.